packages/roosterjs-editor-types/lib/interface/EditorOptions.ts

```diff
@@ -127,4 +128,9 @@ export default interface EditorOptions {
      * @deprecated Use zoomScale instead
      */
     sizeTransformer?: SizeTransformer;
+
+    /**
+     * Retrieves the visible viewport of the Editor. The default viewport is the Rect of the scrollContainer.
+     */
+    getVisibleViewport?: () => Rect;
```

Comment:
I feel with this function, we can deprecate IEditor.getRelativeDistanceToEditor() now since this one is more common and powerful.